local-antora-playbook.yml

```
@@ -15,7 +15,7 @@ content:
   - url: .
     branches: HEAD
   - url: https://github.com/redpanda-data/documentation
-    branches: [main, v/*, api, shared, site-search]
+    branches: ['25.1-rpk-single-source', v/*, api, shared, site-search]
```

```suggestion
    branches: [main, v/*, api, shared, site-search]
```
to commit before merge